run-in-terminal@alextpedro/README.md

```diff
@@ -0,0 +1,46 @@
+![Logo of the project](https://github.com/alextpedro/cinnamon-spices-actions/blob/run-in-terminal%40alextpedro/run-in-terminal%40alextpedro/files/run-in-terminal%40alextpedro/icon.png)
```

Comment:
This should be a link to the upstream repository instead of your fork in case something ever happens to your fork.